src/handler.ts

```diff
@@ -20,18 +20,12 @@ const handler = async (event: DynamoDBStreamEvent, _context: Context, callback:
     const secrets: string[] = await getSecret(process.env.SECRET_NAME);
 
     event.Records.forEach((record) => {
-      if (secrets.includes(record.dynamodb.testStationPNumber as string)) {
-        if (record.dynamodb.testTypeEndTimestamp !== '') {
-          // Send test result
-          const testActivity: TestActivity = dataFormatter(record);
-          logger.info(testActivity);
-        } else {
-          logger.info(
-            `Test result associated with EventID: ${
-              record.eventID
-            } was not completed, event has not been sent to EventBridge`,
-          );
-        }
+      if (secrets.includes(record.dynamodb.NewImage.testStationPNumber.S)) {
```

Comment:
Would it make sense to format the dynamoData earlier so we don't have to worry about the database internals?

Reply:
or at least hide this chain behind a function
if (secrets.includes(extractTestStationNumber(record))) {